src/components/Login/LoginContainer.tsx

```diff
@@ -40,11 +41,14 @@ import { useLogout } from "../../hooks/useLogout";
 import { KeyboardAvoidingScrollView } from "../Layout/KeyboardAvoidingScrollView";
 import * as Linking from "expo-linking";
 import { DOMAIN_FORMAT } from "../../config";
+import { requestOTP, LoginError } from "../../services/auth";
 
 const TIME_HELD_TO_CHANGE_APP_MODE = 5 * 1000;
 
 const ALLOW_MODE_CHANGE = false;
 
+let lastResendWarningMessage = "";
```

Comment:
Why do we still need this variable?

Reply:
it's to store the warning message sent from the backend, because it won't be shown immediately, but only after the user presses resend again